util/omrutil/j9memclr.c

```diff
@@ -113,7 +123,7 @@ OMRZeroMemory(void *ptr, uintptr_t length)
 	memset(ptr, 0, (size_t)length);
 #elif defined(J9ZOS390)
 
-	if (((struct IHAPSA *)0)->FLCFGIEF) {
+	if (1 == pfd_to_zero_object_heap_memory && ((struct IHAPSA *)0)->FLCFGIEF) {
```

Comment:
This could use some extra parens around the `==`, or we could simplify it to:

```suggestion
	if (usePFD && ((struct IHAPSA *)0)->FLCFGIEF) {
```

Reply:
updated!